packages/fast-layouts-react/src/grid/grid.no-css-grid.spec.tsx

```diff
@@ -78,3 +78,85 @@ describe("Grid - without CSS grid support", (): void => {
         expect(rendered.props().style.msGridColumn).toBe(2);
     });
 });
+
+describe("Grid - without CSS grid support but `display` prop is `grid`", (): void => {
```

Comment:
This should be updated to `cssGridPropertyName` (same below)

Reply:
Updated, thanks!